lib/ansible/modules/cloud/ovirt/ovirt_vm.py

```diff
@@ -1630,6 +1629,8 @@ def _attach_cd(self, entity):
 
     def _migrate_vm(self, entity):
         vm_host = self.param('host')
+        if entity.id is None and entity.name is not None:
```

Comment:
Why is this condition needed?

Reply:
I had some issue with the entity but now I tested it now and looks fine without it